lib/streamlit/elements/write.py

```diff
@@ -252,7 +249,7 @@ def write(self, *args: Any, unsafe_allow_html: bool = False, **kwargs) -> None:
             - write(string)         : Prints the formatted Markdown string, with
                 support for LaTeX expression, emoji shortcodes, and colored text.
                 See docs for st.markdown for more.
-            - write(data_frame)     : Displays the DataFrame as a table.
+            - write(data_frame)     : Displays any dataframe-compatible value as table.
```

Comment:
super nit to make it really clear: `as a read-only table` which is the reason why we don't need the data editor

Reply:
Added 👍